pkg/ottl/ottlfuncs/func_contains_test.go

```diff
@@ -0,0 +1,152 @@
+// Copyright The OpenTelemetry Authors
+// SPDX-License-Identifier: Apache-2.0
+
+package ottlfuncs
+
+import (
+	"context"
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.opentelemetry.io/collector/pdata/pcommon"
+
+	"github.com/open-telemetry/opentelemetry-collector-contrib/pkg/ottl"
+)
+
+func Test_contains(t *testing.T) {
+	tests := []struct {
+		name     string
+		target   ottl.StandardPSliceGetter[any]
+		item     ottl.Getter[any]
+		expected bool
+	}{
+		{
+			name: "find item in target",
+			target: ottl.StandardPSliceGetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return []any{"hello", "world"}, nil
+				},
+			},
+			item: ottl.StandardGetSetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return "hello", nil
+				},
+			},
+			expected: true,
+		},
+		{
+			name: "not find item in target",
+			target: ottl.StandardPSliceGetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return []any{"hello", "world"}, nil
+				},
+			},
+			item: ottl.StandardGetSetter[any]{
+				Getter: func(_ context.Context, _ any) (any, error) {
+					return "unknow", nil
```

Comment:
not sure if it was intentional to make this a typo error... you probably meant to write `unknown`